Wikipedia/Code/WMFExploreCollectionViewController.m

```diff
@@ -227,6 +227,10 @@ - (nullable WMFContentGroup *)sectionForIndexPath:(NSIndexPath *)indexPath {
         content = [content wmf_map:^id(WMFFeedNewsStory *obj) {
             return [[obj featuredArticlePreview] articleURL] ?: [[[obj articlePreviews] firstObject] articleURL];
         }];
+    } else if ([group contentType] == WMFContentTypeOnThisDayEvent) {
+        content = [content wmf_map:^id(WMFFeedOnThisDayEvent *obj) {
+            return [[[obj articlePreviews] firstObject] articleURL];
```

Comment:
This can be removed (along with the same code for news story). If you look at where this is used in `presentMoreViewControllerForGroup` - the URLs aren't used for presenting the news vc. You'd also have to remove the assertion there and re-arrange that method a bit.